src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -347,7 +347,7 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 		options?: IConnectionCompletionOptions): Promise<IConnectionResult> {
 		if (options && options.showConnectionDialogOnError) {
 			let params: INewConnectionParams = options && options.params ? options.params : {
-				connectionType: this._connectionStatusManager.isDefaultTypeUri(owner.uri) ? ConnectionType.default : ConnectionType.editor,
+				connectionType: this._connectionStatusManager.isEditorTypeUri(owner.uri) ? ConnectionType.editor : ConnectionType.default,
```

Comment:
What is the URI like in this scenario? Were we not using the `connection` scheme for that?

Reply:
in the scenario of partner team, the scheme is 'dashboard'

Reply:
https://github.com/microsoft/azuredatastudio/pull/17893 this is the PR that I made a couple months ago.